tests/release/pipelines/rh_advisories.go

```diff
@@ -102,7 +102,20 @@ var _ = framework.ReleasePipelinesSuiteDescribe("e2e tests for rh-advisories pip
 			Expect(err).NotTo(HaveOccurred())
 
 			createADVSReleasePlan(advsReleasePlanName, *devFw, devNamespace, advsApplicationName, managedNamespace, "true")
-			component = releasecommon.CreateComponentByCDQ(*devFw, devNamespace, managedNamespace, advsApplicationName, advsComponentName, releasecommon.AdditionalGitSourceComponentUrl)
+			componentObj := appservice.ComponentSpec{
```

Comment:
Does it make sense to have `releasecommon.CreateComponent` as a replacement of `CreateComponentByCDQ`?

Reply:
created a replacement named as `CreateComponent` in place of `CreateComponentByCDQ` , makes the code more clean